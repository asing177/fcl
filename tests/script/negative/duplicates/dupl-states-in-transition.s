transition initial -> a;
transition a -> {a, a}; // should fail duplication check
transition a -> end;
