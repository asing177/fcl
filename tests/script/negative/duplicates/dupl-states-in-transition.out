The declared and actual transitions do not match up:
• Unused: 'transition initial -> a', which is declared but not implemented by any method.
• Unused: 'transition a -> a', which is declared but not implemented by any method.
• Unused: 'transition a -> end', which is declared but not implemented by any method.

Inferred Transitions:
/* No transitions */